cmd/test/test.go

```diff
@@ -0,0 +1,165 @@
+// Copyright 2022 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package test
+
+// See also the Conftest version of this command here:
+// https://github.com/open-policy-agent/conftest/blob/master/internal/commands/test.go
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
+	hd "github.com/MakeNowJust/heredoc"
+	"github.com/open-policy-agent/conftest/output"
+	"github.com/open-policy-agent/conftest/runner"
+	"github.com/spf13/cobra"
+
+	"github.com/enterprise-contract/ec-cli/internal/applicationsnapshot"
+)
+
+const (
+	OutputAppstudio = "appstudio"
+)
+
+var TestCmd *cobra.Command
+
+func fetchTestCmd() *cobra.Command {
+	flags := struct {
+		Policy             []string
+		Data               []string
+		Namespace          []string
+		AllNamespaces      bool
+		NoFail             bool
+		FailOnWarn         bool
+		Output             string
+		NoColor            bool
+		SuppressExceptions bool
+		Trace              bool
+		JunitHideMessage   bool
+	}{}
+
+	cmd := &cobra.Command{
+		Use:   "test [flags] input",
+		Short: "A wrapper for the conftest test command",
+
+		Long: hd.Doc(`
+			The ec test command is meant to work as a drop-in replacement for conftest test.
+
+			It supports most but not all of the flags supported by Conftest. Its supports one
+			more output format than Conftest, the 'appstudio' format, which is suitable for use
+			with the standard AppStudio TEST_OUTPUT Tekton result format.
+		`),
+
+		Example: hd.Doc(`
```

Comment:
It might be a good idea to add an example of using multiple policies and multiple data.